pkg/renew/renew_test.go

```diff
@@ -41,12 +41,18 @@ func TestValidate(t *testing.T) {
 			args:   []string{"abc"},
 			expErr: true,
 		},
+		"If there are arguments, as well as --all-namespaces, error": {
+			options: &Options{
+				AllNamespaces: true,
+			},
+			args:   []string{"abc"},
+			expErr: true,
+		},
 		"If there are all certificates selected, as well as label selector, error": {
 			options: &Options{
 				LabelSelector: "foo=bar",
 				All:           true,
 			},
-			args:   []string{""},
```

Comment:
This looks like a minor bug in the test case, as providing `[]string{""}` as args is triggered a different error `cannot specify Certificate names in conjunction with label selectors` than intended